Tech-Journal-ConsoleApp/Tech-Journal-ConsoleApp/EmailEntry.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Net;
+using System.Net.Mail;
+using System.Text;
+
+namespace Tech_Journal_ConsoleApp
+{
+    class EmailEntry
+    {
+        public const string Path = "c:\\temp\\appsettings.txt";
+
+        private bool _validEmail;
+        private string _toThisEmailAddress;
+        private string _emailUsername;
```

Comment:
After reading this class I think it would provide clarity to change this to `_fromEmailAddress`.
Also you can change the above one to `_toEmailAddress`